### ISSUE TITLE
Subsequent Puppet runs of more than 100 tasks/templates will fail due to API default limit

### ISSUE THREAD

**Issue by druchoo** [m4ce/puppet-kapacitor#2]
If >100 kapacitor::tasks are defined Puppet, initial Puppet run will succeed but subsequent runs will fail:

```
Error: /Stage[main]/Kapacitor/Kapacitor_task[Some_Task_Name]/ensure: change from absent to present failed: Query returned a non successful HTTP code (Code: 400, Error: Bad Request)
Error: Could not run: Query returned a non successful HTTP code (Code: 400, Error: Bad Request)
```

The list tasks and templates methods have a default ```limit``` of 100 which is used by kapacitor-ruby gem. Get call doesn't get all tasks/templates, so some are thought to absent and try to get recreated resulting in 400 error.

https://github.com/influxdata/kapacitor/blob/master/client/API.md#list-tasks

I tested this by setting a limit for ```api_get('/tasks?fields=id')['tasks']``` in kapacitor/client.rb

ex. ```api_get('/tasks?fields=id&limit=125')['tasks']```

**Comment by m4ce** [comment]
Thanks for filing the issue. Right - I may actually be hitting that limit very soon. I will take a look at it shortly.

**Comment by m4ce** [comment]
it appears that this applies to templates too.

**Comment by m4ce** [comment]
I've released this as part of 0.1.1. Give it a try and see if this solves your problem!

**Comment by druchoo** [comment]
Thanks for the fix. Looks good.
